packages/runtime-dom/src/components/Transition.ts

```diff
@@ -154,8 +203,17 @@ export function resolveTransitionProps({
         }
       })
     },
-    onEnterCancelled: finishEnter,
-    onLeaveCancelled: finishLeave
+    onAfterLeave(el) {
+      callHook(onAfterLeave, [el])
+    },
+    onEnterCancelled(el) {
+      finishEnter(el)
+      callHook(onEnterCancelled, [el])
+    },
+    onLeaveCancelled(el) {
+      finishLeave(el)
+      callHook(onLeaveCancelled, [el])
+    }
```

Comment:
This is fix the bug wihch never called `onEnterCancelled ` and `onLeaveCancelled ` hooks